Description: Since monotone defaults to databases ending with .db as
 well, check for files with that extension as well when looking for
 databases.
Origin: upstream, commit: bb5b0702260383c4086ab2088bfa79883f53f6ce
Index: monotone-1.0/extra/shell/monotone.bash_completion.in
===================================================================
--- monotone-1.0.orig/extra/shell/monotone.bash_completion.in	2011-04-16 16:48:56.943628328 +0100
+++ monotone-1.0/extra/shell/monotone.bash_completion.in	2011-04-16 16:49:18.683795516 +0100
@@ -70,6 +70,7 @@
 
 _monotone_databases() {
     _filedir mtn
+    _filedir db
     COMPREPLY=(
 	"${COMPREPLY[@]}"
 	$(compgen -W "$(LANG=POSIX \
